storage/pebble/open.go

```diff
@@ -35,7 +37,7 @@ func NewBootstrappedRegistersWithPath(dir string) (*Registers, *pebble.DB, error
 
 // OpenRegisterPebbleDB opens the database
```

Comment:
can you update these comments to explain the difference between this and `OpenDefaultPebbleDB`